test/test.gypi

```diff
@@ -62,7 +62,6 @@
         'resources/mock_file_source.cpp',
         'resources/mock_file_source.hpp',
         'resources/mock_view.hpp',
-        'resources/resource_loader.cpp',
```

Comment:
I'm rebasing as part of #1674.

BTW, awesome job, merging and moving the stuff to Style made the utest a lot cleaner.